src/clr-angular/forms/styles/_file.clarity.scss

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+.clr-control-container {
+    .clr-file-wrapper {
+        @include form-flatten-validate-text();
+        position: relative;
+        margin-top: $clr-forms-baseline;
+
+        .clr-file {
+            width: 0.1px;
```

Comment:
wouldn't `height: 0` and `width: 0` work?

Reply:
looks like, some research I saw said it had to be over 0, but need to verify cross-browser